frontend/src/metabase/lib/colors/palette.ts

```diff
@@ -91,7 +91,7 @@ export function color(color: any, palette: ColorPalette = colors) {
     ...palette,
   };
 
-  if (color in palette) {
+  if (color in fullPalette) {
```

Comment:
The previous version was wrong because `palette` is partial, not full palette, so it could have less keys than `colors`